### PR TITLE
f-form-field@v1.5.0 – Updating data-test-id and tidy-up of styling

### PR DESCRIPTION
**Pull request by ashleynolan**
### Changed
- `data-test-id`'s updated across `FormField` and `FormDropdown` components.
- Updating CSS variable names and classnames.
- Updated `isInputField` to be called `isSelectionControl`.

### Fixed
- Extra space removed in `option` tag of dropdown component.

---

## UI Review Checks

- [x] Unit tests have been updated

## Browsers Tested

- [x] Chrome (latest)

### CODE REVIEW

File: packages/f-form-field/src/components/FormField.vue [hunk 1]
```diff
@@ -56,7 +53,7 @@
                 :label-style="normalisedLabelStyle"
                 :for="uniqueId"
                 :is-inline="isInline"
-                data-test-id="form-field-label--inline">
+                :data-test-id="testId.label + '--inline'">
```
Comment (kevinrodrigues) [inline packages/f-form-field/src/components/FormField.vue]: Could use a template string here.

File: packages/f-form-field/src/components/FormField.vue [hunk 2]
```diff
@@ -179,7 +171,13 @@ export default {
         },
 
         testId () {
-            return this.dataTestId || this.$attrs.name || false;
+            const formFieldName = this.$attrs.name;
```
Comment (kevinrodrigues) [inline packages/f-form-field/src/components/FormField.vue:173]: jsDocs for the two methods that have changed?

File: packages/f-form-field/src/components/tests/FormField.test.js [hunk 3]
```diff
@@ -155,4 +155,66 @@ describe('FormField', () => {
             });
         });
     });
+
+    describe('attrs ::', () => {
+        describe('name ::', () => {
+            it('when name is not set, testId.container should be set to `formfield-container`', () => {
```
Comment (kevinrodrigues) [inline packages/f-form-field/src/components/tests/FormField.test.js]: Could these all start with `should` please? `should set testId. container  to 'formfield-container' when name is not set`

File: packages/f-form-field/src/components/FormField.vue (hunk 1 above)
Comment (ashleynolan) [reply packages/f-form-field/src/components/FormField.vue]: Fixed

File: packages/f-form-field/src/components/FormField.vue (hunk 2 above)
Comment (ashleynolan) [reply packages/f-form-field/src/components/FormField.vue:173]: Can I raise a separate PR straight after documenting all the functions in f-form-field? Will do them all at the same time then.

File: packages/f-form-field/src/components/tests/FormField.test.js (hunk 3 above)
Comment (ashleynolan) [reply packages/f-form-field/src/components/tests/FormField.test.js]: Good suggestion – have updated

**Review by FayeCarter — APPROVED** [review]

**Review by kevinrodrigues — APPROVED** [review]

**Review by siggerzz — APPROVED** [review]

**Pull request merged**